py3status/modules/battery_level.py

```diff
@@ -225,6 +219,26 @@ def _refresh_battery_info(self):
         Battery 1: Unknown, 98%
         Battery 1: design capacity 1879 mAh, last full capacity 1370 mAh = 72%
         '''
+
+        def _parse_battery_info(acpi_battery_lines):
+            battery = dict()
+            battery["percent_charged"] = int(findall("(?<= )(\d+)(?=%)",
+                                                     acpi_battery_lines[0])[0])
+            battery["charging"] = "Charging" in acpi_battery_lines[0]
+            battery["capacity"] = int(findall("(?<= )(\d+)(?= mAh)",
+                                              acpi_battery_lines[1])[1])
+
+            # ACPI only shows time remaining if battery is discharging or
+            # charging
+            try:
+                battery["time_remaining"] = ''.join(findall(
+                    "(?<=, )(\d+:\d+:\d+)(?= remaining)|"
+                    "(?<=, )(\d+:\d+:\d+)(?= until)", acpi_battery_lines[0])[0])
+            except IndexError:
+                battery["time_remaining"] = '?'
```

Comment:
I'd be inclined to have the `'?'` as a constant in case we ever decide to change it, as it is used in 2 places and that would make the change easier imho